checkstyle-tester/diff.groovy

```diff
@@ -289,6 +291,27 @@ def generateSummaryIndexHtml(diffDir, checkstyleBaseReportInfo, checkstylePatchR
     println 'Creating report summary page finished...'
 }
 
+def generateListOfProjectsHtml(diffDir, listOfProjects) {
+   def diffToolDir = Paths.get("").toAbsolutePath()
+        .parent
+        .resolve("patch-diff-report-tool")
+        .toFile()
+    def diffToolJarPath = getPathToDiffToolJar(diffToolDir)
+    this.class.classLoader.rootLoader.addURL(new URL("file:$diffToolJarPath"))
+    def textTransform = Class.forName("com.github.checkstyle.site.TextTransform").newInstance()
+    def listOfProjectsHtml = new File("$diffDir/listofprojects.html")
+    textTransform.transform(listOfProjects, listOfProjectsHtml.toPath().toString(), Locale.ENGLISH,
+        "UTF-8", "UTF-8")
+
+    return listOfProjectsHtml
```

Comment:
Shouldn't this return the generated HTML for the page instead of the file?

Reply:
@nmancus1 I feel this point stands why return the file and just go directly to printing? Why not combined `generateConfigFileHtml` and `printConfigInfo` into one?

Reply:
Fixed.